internal/delivery/http/v1/controller/matches/handler/create_match_handler.go

```diff
@@ -0,0 +1,40 @@
+package matchHandler
+
+import (
+	"cats-social/internal/delivery/http/v1/request"
+	"cats-social/internal/delivery/http/v1/response"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/golang-jwt/jwt/v4"
+)
+
+func (mh matchHandler) CreateMatch(c *fiber.Ctx) error {
+	var (
+		req  request.CreateMatch
+		resp *response.CreateMatch
+		err  error
+	)
+	err = c.BodyParser(&req)
+	if err != nil {
+		return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{
+			"error": err.Error(),
+		})
+	}
+
```

Comment:
Kyknya bisa ditambahin validasi disini:
```suggestion
// Create a new validator instance
	validate := validator.New()

	// Validate the Create Match struct
	err = validate.Struct(req)
	if err != nil {
		// Validation failed, handle the error
		return lumen.FromError(lumen.NewError(lumen.ErrBadRequest, err)).SendResponse(c)
	}
```